dist/index.html

```diff
@@ -32,7 +32,9 @@ <h1 class='travel-title'>travel.</h1>
 
     </nav>
     <section class='book-trip'>
```

Comment:
check your consistency between single and double quoties

Reply:
thank you - making the changes on my local repo!